conftest.py

```diff
@@ -114,3 +117,82 @@ def image_xr(rng: np.random.Generator, library: pd.DataFrame) -> Generator[xr.Da
         dims=["peak", "x", "y"],
     )
     yield img_xr
+
+
+@pytest.fixture(scope="session")
+def glycan_img_path(tmp_path_factory: TempPathFactory, imz_data: ImzMLParser, rng: np.random.Generator):
+    coords: np.ndarray = np.array([coord[:2] for coord in imz_data.coordinates])
+
+    glycan_img: np.ndarray = np.zeros((10, 10))
+    glycan_img[coords[:, 1] - 1, coords[:, 0] - 1] = rng.random(coords.shape[0])
+
+    glycan_img_file: Path = tmp_path_factory.mktemp("glycan_imgs") / "glycan_img.tiff"
+    io.imsave(glycan_img_file, glycan_img)
+
+    yield glycan_img_file
+
+
+@pytest.fixture(scope="session")
+def poslog_path(tmp_path_factory: TempPathFactory, imz_data: ImzMLParser):
+    columns_write: List[str] = ["Date", "Time", "Region", "PosX", "PosY", "X", "Y", "Z"]
+    poslog_data: pd.DataFrame = pd.DataFrame(
+        np.random.rand(len(imz_data.coordinates) + 2, len(columns_write)), columns=columns_write
```

Comment:
I believe `np.random.default_rng` is the recommended way to generate random numbers in NumPy as of late.

We can use the `rng` fixture to generate the random data 

```python
rng.random(size=(len(imz_data.coordinates) + 2, len(columns_write))]
```